internal/samsahai/rpc.go

```
@@ -876,3 +901,33 @@ func (c *controller) sendPullRequestTriggerReport(prTrigger *s2hv1.PullRequestTr
 		}
 	}
 }
+
+func (c *controller) sendPullRequestTestRunnerPendingReport(prQueue *s2hv1.PullRequestQueue, teamWithPR *rpc.TeamWithPullRequest) error {
+	configCtrl := c.GetConfigController()
+
+	teamComp := &s2hv1.Team{}
+	err := c.getTeam(teamWithPR.TeamName, teamComp)
+	if err != nil {
+		logger.Error(err, "cannot get team", "team", teamWithPR.TeamName)
+		return err
+	}
+
+	if err := c.LoadTeamSecret(teamComp); err != nil {
+		logger.Error(err, "cannot load team secret", "team", teamComp.Name)
+		return err
+	}
+
+	bundleName := prQueue.Spec.BundleName
+	prNumber := prQueue.Spec.PRNumber
+	commitSHA := prQueue.Spec.CommitSHA
+	for _, reporter := range c.reporters {
+		prTriggerRpt := s2h.NewPullRequestTestRunnerPendingReporter(c.configs, teamWithPR.TeamName,
+			bundleName, prNumber, commitSHA, teamComp.Status.Used.Credential)
+
+		if err := reporter.SendPullRequestTestRunnerPendingResult(configCtrl, prTriggerRpt); err != nil {
+			logger.Error(err, "cannot send pull request trigger result report",
+				"team", teamWithPR.TeamName, "bundle", bundleName, "prNumber", prNumber)
+		}
```

Could we return the err after log as well?